src/test/java/uk/gov/hmcts/reform/sendletter/data/LetterRepositoryTest.java

```diff
@@ -146,4 +149,26 @@ public void should_return_empty_optional_case_when_no_letters_found() {
 
         assertThat(result.orElse(null)).isNull();
     }
+
```

Comment:
Could you also add a test case where there are zero not printed letters?

Reply:
Was thinking about it but in a way - does really make sense? Will also cover that case with functional test

Reply:
If you are going to add functional test case then I think that is fine 👍